.github/workflows/pull_request.yml

```
@@ -8,3 +8,15 @@ jobs:
   tests:
     name: Test
     uses: swiftlang/github-workflows/.github/workflows/swift_package_test.yml@main
+  soundness:
+    name: Soundness
+    uses: swiftlang/github-workflows/.github/workflows/soundness.yml@main
+    with:
+      api_breakage_check_enabled: true
+      broken_symlink_check_enabled: true
+      docs_check_enabled: true
+      format_check_enabled: true
```

```suggestion
      format_check_enabled: false
```

For now disable the format check until the repo is updated.